cmd/chartmuseum/main.go

```diff
@@ -231,6 +242,21 @@ var cliFlags = []cli.Flag{
 		Usage:  "prefix to store charts for --storage-google-bucket",
 		EnvVar: "STORAGE_GOOGLE_PREFIX",
 	},
+	cli.StringFlag{
+		Name:   "storage-azure-name",
+		Usage:  "name to store charts for --storage-azure-name",
+		EnvVar: "STORAGE_AZURE_NAME",
+	},
+	cli.StringFlag{
+		Name:   "storage-azure-key",
+		Usage:  "key to store charts for --storage-azure-key",
+		EnvVar: "STORAGE_AZURE_KEY",
+	},
+	cli.StringFlag{
+		Name:   "storage-azure-container",
+		Usage:  "container to store charts for --storage-azure-container",
+		EnvVar: "STORAGE_AZURE_KEY",
```

Comment:
I believe this should be STORAGE_AZURE_CONTAINER

Reply:
Correct, thanks!